code/modules/events/ion_storm.dm

```diff
@@ -566,6 +566,35 @@
 							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
 
 	return message
+	
+
+/proc/corp_name()

```

Comment:
That makes no sense.
@optimumtact this is the file for the ion storm EVENT.
Why should this go here?